website/docs/reference/widgets/README.md

```diff
@@ -24,30 +24,73 @@ You can access the various properties of the widget using the widget's name.
 {{ Table1.selectedRow.id }}
 ```
 
-### Grouping Widgets
+### Grouping widgets
 
-Appsmith supports the grouping of widgets. When you group widgets, they are put in a container and can be moved together. To do this -
+Appsmith supports the grouping of widgets. When you group widgets, they're put in a container and can be moved together. To do this -
 
 * Select multiple widgets with Ctrl + Left Click
-* Now click on the dotted square icon or press Ctrl + G
+* Now click in the dotted square icon or press Ctrl + G
 
 ![](/img/groupingwidget.gif)
 
-### Common Properties
+### Common properties
 
 The following properties are common across many of Appsmith's widgets. You can find them by selecting your widget and checking its properties pane, and you can use them to customize the details and behavior of your app.
 
 | Property            | Description                                                                                                                                                                                      |
 | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [**Height**](#height)        | It configures how a widget’s height reacts to content changes. It has three possible configurations:<br/>**[Fixed](#fixed)**: The height of the widget remains as set using drag and resize.<br/>**[Auto Height](#auto-height)**: The height of the widget reacts to content changes.<br/>  **[Auto Height with limits](#auto-height-with-limits)**: Same as Auto height, with a configurable option to set the minimum and maximum number of rows that can be occupied by the widget.                                      |
```

Comment:
Keep this description small. no need to mention the three configurations. because that is already there in the details.